apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -111,19 +129,26 @@ const CreateEventPage = ({ history }) => {
         .required('Event Description is required.')
         .max(5000, 'Event Description must be under 5000 characters.'),
       location: Yup.string().required('Event Location is required.'),
+      imageFile: Yup.mixed().required('Image File is required.'),
     }),
 
-    onSubmit: (values) => {
+    onSubmit: async (values) => {
       notifySubmissionToast();
       if (!values.endDate) values.endDate = cloneDeep(values.startDate);
       addTimeToDate(values.startTime, values.startDate);
       addTimeToDate(values.endTime, values.endDate);
+      let media = await uploadFile({ variables: { file: file } });
+      let imageMedia: MediaType = {
```

Comment:
```suggestion
      const media = await uploadFile({ variables: { file: file } });
      const imageMedia: MediaType = {
```
Since these are never reassigned.